src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.ts

```diff
@@ -98,10 +97,6 @@ export class RgwUserDetailsComponent implements OnChanges, OnInit {
     }
   }
 
-  updateKeysSelection(selection: CdTableSelection) {
-    this.keysSelection = selection;
-  }
-
```

Comment:
why's this getting removed here?

Reply:
I think merge conflict created this, I'll remove.